packages/eas-cli/src/commands/release/list.ts

```diff
@@ -0,0 +1,83 @@
+import { Command, flags } from '@oclif/command';
+import CliTable from 'cli-table3';
+import gql from 'graphql-tag';
+
+import { graphqlClient, withErrorHandlingAsync } from '../../graphql/client';
+import { RootQuery, UpdateRelease } from '../../graphql/generated';
+import log from '../../log';
+import { findProjectRootAsync, getProjectFullNameAsync } from '../../project/projectUtils';
+
+async function listReleasesAsync({ fullName }: { fullName: string }): Promise<UpdateRelease[]> {
+  const data = await withErrorHandlingAsync(
+    graphqlClient
+      .query<RootQuery>(
+        gql`
+          query ReleasesByAppQuery($fullName: String!) {
+            app {
+              byFullName(fullName: $fullName) {
+                id
+                fullName
+                updateReleases(offset: 0, limit: 10000) {
```

Comment:
What is the plan for if there are more than 10,000? Should we log a warning if the number of releases is >= 10,000? `Returning the max number of releases, 10,000`